lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,25 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bazel"
```

Comment:
I've only been able to find 88 `.bazel` files that are not `BUILD.bazel` and none of them seem to be Starlark files. I'd prefer that we keep `BUILD.bazel` as a filename and remove this extension.

Reply:
Sure thing. Adding `WORKSPACE.bazel` as well since [it is supported](https://docs.bazel.build/versions/master/build-ref.html#workspace).

Reply:
I think we're too low on usage for `WORKSPACE.bazel`. There's a [single such file](https://github.com/search?q=filename%3AWORKSPACE.bazel&type=Code) on GitHub.

Reply:
Bazel maintainer here. We just added support for this in the core build system, so it's pretty fresh. It'll be nice to have this though.

Reply:
I can add it to #4219 to monitor usage then.